### DISCUSSION TITLE
Heimdall : Add option to select port number

### DISCUSSION THREAD

**Discussion by fabiengagne** [tteck/Proxmox#3865] [Ideas]
Currently the script installs Heimdall to serve port 7990. It would be useful to have it installed to port 80 by default, or provide an option to select the port number in the advanced options.

**Comment by fabiengagne** [comment]
In the meantime, I've changed the port in the service script after installation.
```
nano /etc/systemd/system
```
(changed 7990 to 80)
```
systemctl daemon-reload
systemctl restart heimdall
```

**Comment by fabiengagne** [comment]
@tteck I see that you have closed this item with no further comments but I don't see any modifications to ct/heimdall* or install/heimdall*, at least not as of today.

**Reply by tteck** [reply]
The user can modify the service file if needed. I will not change the code to make it changeable.

**Comment by fabiengagne** [comment]
Ok then the default should be port 80, as in the [dockerhub](https://hub.docker.com/r/linuxserver/heimdall)
